### PR TITLE
Make mne/digitization private mne/_digitization

### PR DESCRIPTION
**Pull request by massich**
Do not review. this goes after #6639

### CODE REVIEW

**Comment by massich** [comment]
Does anyone know why the CIs do not do what I expect??? they are not running.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ Changelog
 
 - Speed up morph map generation in :func:`mne.read_morph_map` by ~5-10x by using :func:`numba.jit` by `Eric Larson`_
 
-- Add :class:`mne.digitization.Digitization` class to simplify montage by `Joan Massich`_
+- Add ``Digitization`` class to simplify montage by `Joan Massich`_
```
Comment (agramfort) [inline doc/whats_new.rst]: can you now remove this entry?

Comment (massich) [reply doc/whats_new.rst]: the whole thing? sure.

**Comment by agramfort** [comment]
yes. Digitization is not anything that should be advertised anymore.

>

**Comment by massich** [comment]
closed in favor of #6700 (I've no idea why was not working. moving on!)

**Pull request closed without merging**